core/src/templates/components/card/card.twig

```diff
@@ -50,31 +62,31 @@
   {{ card_body }}
 
   {# Card CTA Link #}
-  {%- if card_cta_label is not empty %}
+  {%- if card_cta_label is not empty and allow_links %}
```

Comment:
This is a nice little helper protector.

Reply:
Yeah, that's from @JBCSU 's twig tweak. I just did `if "su-card--link" is not in modifier_class` before, but this is cleaner.